README_v4.md

```diff
@@ -176,6 +176,23 @@ Please take a look at the `tests` folder, you'll find examples of each possible
       });
     });
     ```
+* **Running code after every test**: just like many testing frameworks have, there is a way to execute some code after (cleanp) every test in a suite using the `after()` function. Example:
```

Comment:
:thought_balloon: thank you for adding docs! technically this won't be available in version 4 so I think we should add this section only to the main `README`

Reply:
done